src/smk/viewer-leaflet/viewer-leaflet.js

```diff
@@ -29,6 +29,9 @@ include.module( 'viewer-leaflet', [ 'viewer', 'leaflet', 'layer-leaflet', /*'fea
             minZoom:            smk.viewer.minZoom
         } )
 
+        self.map.createPane('basemaps');
```

Comment:
Could you please add a comment explaining what a 'pane' is and why this is necessary (for future reference).